src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/multiplestringliterals/Example1.java

```diff
@@ -6,14 +6,17 @@
 </module>
 */
 
+package com.puppycrawl.tools.checkstyle.checks.coding.multiplestringliterals;
+
 // xdoc section -- start
-public class MyClass {
-  String a = "StringContents";
+public class Example1 {
+  String a = "StringContents"; // violation, "StringContents" occurs twice
   String a1 = "unchecked";
   @SuppressWarnings("unchecked") // OK, duplicate strings are ignored in annotations
   public void myTest() {
-    String a2 = "StringContents"; // violation, "StringContents" occurs twice
-    String a3 = "DoubleString" + "DoubleString"; // violation, "DoubleString" occurs twice
+    String a2 = "StringContents"; // OK
```

Comment:
Please remove all comments at are just `// OK` .
We allow ok comments with details

Reply:
done